src/ui/reducers/app.ts

```diff
@@ -115,9 +114,7 @@ const appSlice = createSlice({
     updateTheme(state, action: PayloadAction<AppTheme>) {
       state.theme = action.payload;
     },
-    setInitializedPanels(state, action: PayloadAction<PanelName>) {
-      state.initializedPanels.push(action.payload);
-    },
```

Comment:
really happy to be able to kill this. It's an example of some of the weird classes that predated and then lived alongside the react apps